src/File.php

```diff
@@ -157,7 +157,10 @@ class File extends DataObject implements AssetContainer, Thumbnail, CMSPreviewab
     );
 
     private static $casting = array (
-        'TreeTitle' => 'HTMLFragment'
+        'TreeTitle' => 'HTMLFragment',
+        'getTreeTitle' => 'HTMLFragment',
+        'Tag' => 'HTMLFragment',
+        'getTag' => 'HTMLFragment',
```

Comment:
Is `getTag` as well as `Tag` required? Perhaps we should fix that (separate to this PR) if so 😕

Reply:
It should really just be the short name ('Tag') but it doesn't hurt I guess?